lighthouse-core/gather/gatherers/dobetterweb/stacks.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/**
+ * @fileoverview Gathers a list of detected JS libraries and their versions.
+ */
+
+'use strict';
+
+const JSLibraries = require('./js-libraries');
+
+class Stacks extends JSLibraries {
```

Comment:
I created a new gatherer as renaming JSLibraries would be a breaking change.

Reply:
Plugins isn't shipped yet, so IMO our artifacts are not yet part of the public API. 
We can just rename the artifact.

Reply:
isn't this part of lhr yet? if so, this would be a breaking change for consumers of the json.

Reply:
artifacts aren't put into the LHR so I think we're good here until plugins are an official shipped *thing*™

Reply:
thanks for the reminder so I just rename JSLibraries

Reply:
[https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/dobetterweb/js-libraries.js](js-libraries audit) is using [JSLibraries](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/gather/gatherers/dobetterweb/js-libraries.js) and need the extra information that it provides